examples/js/modules/pagination.js

```diff
@@ -0,0 +1,132 @@
+const range = document.getElementById('range');
+const prevBtn = document.getElementById('prevBtn');
+const nextBtn = document.getElementById('nextBtn');
+
+export class Paginate {
+  constructor(url, count, fetchedImages, handleNext, handlePrev) {
+    this.url = url;
+    this.count = count;
+    this.fetchedImages = fetchedImages;
+    nextBtn.addEventListener('click', handleNext);
+    prevBtn.addEventListener('click', handlePrev);
+  };
+
+  paginate(images) {
+    const imgsPerPage = 100;
+    const pages = Math.ceil(images.length / imgsPerPage);
+    const paginatedImages = Array.from({length: pages}, (_, index) => {
+      const start = index * imgsPerPage;
+      return images.slice(start, start + imgsPerPage);
+    });
+    return paginatedImages;
```

Comment:
So here, we'd create an internal persistent variable (like 'range' on line 1), and we'd store this state internally.

Reply:
That's true, I never looked at it this way